tox.ini

```diff
@@ -37,4 +37,4 @@ ignore = E203,E501,W503,E231,A003,D101,D102,D103,D104,D105,D107,A003,D100
 
 [testenv:docs]
 deps = sphinx
-commands = python setup.py build_sphinx
\ No newline at end of file
+commands = sphinx-build -b html doc/source doc/build/html
```

Comment:
does not seems related?

Reply:
Sphinx 7 has removed the setuptools integration https://github.com/sphinx-doc/sphinx/pull/11363
`setup.py` has been long deprecated and projects are encouraged to migrate away from using it.

Reply:
Ok, we should make that in a different PR though.

Reply:
PR proposed https://github.com/Mergifyio/daiquiri/pull/76